src/backend/app/submissions/submission_routes.py

```diff
@@ -64,6 +64,7 @@ async def read_submissions(
 @router.get("/download")
 async def download_submission(
     project_user: Annotated[ProjectUserDict, Depends(mapper)],
+    _: Annotated[bool, Depends(project_contributors)],
```

Comment:
We don't need two dependency here, we can simple use project_contributor instead of mapper.
```python
project_user: Annotated[ProjectUserDict, Depends(project_contributors)]
like this.